pkg/kubelet/apis/pluginregistration/v1alpha/constants.go

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package pluginregistration
+
+const (
+	CSIPlugin      = "CSIPlugin"
+	DevicePlugin   = "DevicePlugin"
+	PluginsSockDir = "/var/lib/kubelet/plugins/"
```

Comment:
I found "/var/lib/kubelet/plugins/" could already been used to store persistent plugin data:
https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/kubelet_getters.go#L56
with kl.getRootDir() usually set to "/var/lib/kubelet" on a lot distros. For now, should we define this as "/var/lib/kubelet/plugins_sockets/" to avoid potential conflicts? In the future, we will need to think about how to make this path specification more portable, as we have brainstormed in the meeting.

Reply:
Let's not hardcode directories. Instead let's derive this from kubelet's root directory. This means that plugins will have to be dynamically configured if necessary in certain environments, but that's a separate problem to be solved.

Reply:
Sounds reasonable. We can add a similar getter function on kubelet side to get this directory. On plugin side, they can set the path with a flag.